tests/test_services/test_notify.py

```diff
@@ -227,6 +227,52 @@ def test_send_submit_application_email(self, app):
         assert resp == Notification(id=uuid.UUID("00000000-0000-0000-0000-000000000003"))
         assert request_matcher.call_count == 1
 
+    @responses.activate
+    def test_send_submit_application_email_welsh(self, app):
+        request_matcher = responses.post(
+            url="https://api.notifications.service.gov.uk/v2/notifications/email",
+            status=201,
```

Comment:
maybe this should be a parameterised test instead?

Reply:
Done